src/shadowbox/server/run_action.sh

```diff
@@ -14,20 +14,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo "** Currently broken, use yarn do shadowbox/docker/run instead **" >&2
-exit 1
-
 do_action shadowbox/server/build
 
+readonly RUN_ID="${RUN_ID:-$(date +%Y-%m-%d-%H%M%S)}"
+readonly RUN_DIR="/tmp/outline/$RUN_ID"
+echo "Using directory $RUN_DIR"
+
 export LOG_LEVEL="${LOG_LEVEL:-debug}"
 export SB_PUBLIC_IP="${SB_PUBLIC_IP:-$(curl https://ipinfo.io/ip)}"
 # WARNING: The SB_API_PREFIX should be kept secret!
 export SB_API_PREFIX=TestApiPrefix
 export SB_METRICS_URL=https://dev.metrics.getoutline.org
-export SB_STATE_DIR=/tmp/outline
+export SB_STATE_DIR="${RUN_DIR}/persisted-state"
+export SB_BIN_DIR="${RUN_DIR}/bin"
```

Comment:
No need to specify bin dir. See Dockerfile comment.

Reply:
Done.